mlflow/store/sftp_artifact_repo.py

```diff
@@ -41,8 +41,14 @@ def __init__(self, artifact_uri, client=None):
             if 'hostname' in user_config:
                 self.config['host'] = user_config['hostname']
 
-            if self.config.get('username', None) is None and 'username' in user_config:
-                self.config['username'] = user_config['username']
+            if self.config.get('username', None) is None and 'user' in user_config:
```

Comment:
Looks like you change this to look for 'user' instead of 'username'. Shouldn't it look for 'username'?

Reply:
yes, I'm pretty sure we should look for 'user'. we can find the documents in [Linux man page](https://linux.die.net/man/5/ssh_config)

Reply:
OK, thanks. This looks good then.